### ISSUE TITLE
Error when using the default config file

### ISSUE THREAD

**Issue by bulutcambazi** [pokt-network/relay_counter#4]
I am receiving this error on healthy nodes, completely up to speed with current block height and with bunch of relays and rewards. 

This error doesn't happen if I use blocks as timeline unit vs. days. Perhaps a bug in binary search logic?

```
go run ./ ...
2021/02/23 04:03:48 Attempting to read Config file:
2021/02/23 04:03:48 Config File Processed:
2021/02/23 04:03:48 {{-4 -1 days} http://localhost:8082/v1 3 {15 4}}
2021/02/23 04:03:48 Testing Pocket Endpoint
2021/02/23 04:03:48 Converting Timeline To Block Heights
2021/02/23 04:03:48 Getting the latest height
2021/02/23 04:03:48 Getting the latest block
2021/02/23 04:03:48 Latest height is 19243 and latest time is: 2021-02-23 03:41:33.185792419 +0000 UTC
2021/02/23 04:03:48 Timeline unit is days
2021/02/23 04:03:48 Calclating the approximate start and end times
2021/02/23 04:03:48 Target start: 2021-02-19 03:41:33.185792419 +0000 UTC
Target End: 2021-02-22 03:41:33.185792419 +0000 UTC
2021/02/23 04:03:48 Begin Closest Height Operations
2021/02/23 04:03:48 Performing a binary search for the closest height to the target time: 2021-02-19 03:41:33.185792419 +0000 UTC
2021/02/23 04:03:48 min:  0 max 19243 try height 18859 closest height 18859
2021/02/23 04:03:48 min:  18859 max 19243 try height 19051 closest height 18859
2021/02/23 04:03:48 RPC failure for block by height: Unregistered interface types.Evidence
Trying to retry. Retry count is: 0/3
2021/02/23 04:03:53 min:  18859 max 19243 try height 19051 closest height 18859
2021/02/23 04:03:54 RPC failure for block by height: Unregistered interface types.Evidence
Trying to retry. Retry count is: 1/3
2021/02/23 04:03:59 min:  18859 max 19243 try height 19051 closest height 18859
2021/02/23 04:03:59 RPC failure for block by height: Unregistered interface types.Evidence
Trying to retry. Retry count is: 2/3
2021/02/23 04:04:04 min:  18859 max 19243 try height 19051 closest height 18859
2021/02/23 04:04:04 After 3 retries, unable to get block for height: 19051, with error: Unregistered interface types.Evidence
exit status 1
```

**Comment by andrewnguyen22** [comment]
Hey @bulutcambazi thanks for the log file,

This error is common when using a Pocket Core Node version < RC-0.5.2.9 due to a Tendermint incompatibility, can you confirm that you are getting this error on RC-0.5.2.9 or higher?

Thanks

**Comment by bulutcambazi** [comment]
Yes, it is 5.2.9.

**Comment by bulutcambazi** [comment]
Also wanted to add that I tried this on two different nodes, both healthy and fully synced.

**Comment by andrewnguyen22** [comment]
Cool I'll fix it

**Comment by andrewnguyen22** [comment]
closed with f581278

**Comment by andrewnguyen22** [comment]
@bulutcambazi let me know if this fixed it for you
